Do not use the _libxml_check_lib() on cross-compile

Upstream-Status: Inappropriate [configuration] 

xml2 have been added into package's DEPENDS, so not need to use the
_libxml_check_lib() to check it again, and _libxml_check_lib() always
return false on cross-compile environment

Signed-off-by: Roy Li <rongqing.li@windriver.com>
---
 Makefile.PL |   52 ++++++++++++++++++++++++++--------------------------
 1 file changed, 26 insertions(+), 26 deletions(-)

diff --git a/Makefile.PL b/Makefile.PL
index c0485f1..09c676b 100644
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -411,32 +411,32 @@ sub _libxml_check_lib {
     }
 }
 
-print "Checking for ability to link against xml2...";
-if ( _libxml_check_lib('xml2') ) {
-    print "yes\n";
-}
-else {
-    print "no\n";
-    print "Checking for ability to link against libxml2...";
-    if ( _libxml_check_lib('libxml2')) {
-        print "yes\n";
-    }
-    else {
-        print STDERR <<"DEATH";
-libxml2, zlib, and/or the Math library (-lm) have not been found.
-Try setting LIBS and INC values on the command line
-Or get libxml2 from
-  http://xmlsoft.org/
-If you install via RPMs, make sure you also install the -devel
-RPMs, as this is where the headers (.h files) are.
-
-Also, you may try to run perl Makefile.PL with the DEBUG=1 parameter
-to see the exact reason why the detection of libxml2 installation
-failed or why Makefile.PL was not able to compile a test program.
-DEATH
-        exit 0; # 0 recommended by http://cpantest.grango.org (Notes for CPAN Authors)
-    }
-}
+#print "Checking for ability to link against xml2...";
+#if ( _libxml_check_lib('xml2') ) {
+#    print "yes\n";
+#}
+#else {
+#    print "no\n";
+#    print "Checking for ability to link against libxml2...";
+#    if ( _libxml_check_lib('libxml2')) {
+#        print "yes\n";
+#    }
+#    else {
+#        print STDERR <<"DEATH";
+#libxml2, zlib, and/or the Math library (-lm) have not been found.
+#Try setting LIBS and INC values on the command line
+#Or get libxml2 from
+#  http://xmlsoft.org/
+#If you install via RPMs, make sure you also install the -devel
+#RPMs, as this is where the headers (.h files) are.
+#
+#Also, you may try to run perl Makefile.PL with the DEBUG=1 parameter
+#to see the exact reason why the detection of libxml2 installation
+#failed or why Makefile.PL was not able to compile a test program.
+#DEATH
+#        exit 0; # 0 recommended by http://cpantest.grango.org (Notes for CPAN Authors)
+#    }
+#}
 
 # -------------------------------------------------------------------------- #
 # _NOW_ write the Makefile
-- 
1.7.10.4

